ods_ci/tests/Tests/1300__model_registry/Model_Registry_Testing.md

```diff
@@ -0,0 +1,29 @@
+# Model Registry ODS-CI Testing
+This README is intended to demonstrate how to run the Model Registry ODS-CI Robot Framework tests in a local environment.
+## Requirements
+Before proceeding please ensure you have read and followed this document [Link Text](../../../../README.md)
+## Local Deployment of Model Registry Secure DB Test
+This test applies the SecuredDB version of model regisrty to an Istio/ODH/ModelMesh cluster.
+To review and run it locally you will need the following:
+
+- Have an existing openshift cluster with Red Hat Athorino, OpenDataHub and Red Hat Openshift Service Mesh operators installed.
+
+- Be oc logged into to your cluster
+
+- Create and prepare the test-variables.yml file by using the template file test-variables.yml.example file. The following will need to be set.
+
+  OCP_CONSOLE_URL: "https://console-openshift-console.<YOUR_DOMAIN>.com/"
+  
+  ODH_DASHBOARD_URL: "https://odh-dashboard-opendatahub.<YOUR_DOMAIN>.com/"
+
+- Set the following local files
+  
+  ods_ci/run_robot_test.sh  line 25 SUBFOLDER to be set to true.
```

Comment:
what does it mean?

Reply:
ah, but you should use `--no-output-subfolder true` script argument instead of changing the script code

Reply:
This is a README for the guys in model-registry to help them use the test locally.  So, if I'm getting this I could pass --no-output-subfolder true as an argument in the run command?